server/views/components/iframed-video/index.njk

```diff
@@ -1,3 +1,4 @@
 <div class="iframed-video">
     <iframe src="{{ video.embedUrl }}" frameborder="0" class="iframed-video__iframe"></iframe>
 </div>
+{{ video.description }}
```

Comment:
In order to make it responsive, the iframe is absolutely positioned and sized to match it's parents dimensions.  So will need another containing div to house the video and description. 

Will revisit this whole thing, with Tomi's designs in hand